en/identity-server/7.1.0/docs/apis/organization-apis/restapis/notification-senders.yaml

```diff
@@ -0,0 +1,1363 @@
+          example: smtp.gmail.com
+        smtpPort:
+          type: integer
+          example: 587
+        fromAddress:
+          type: string
+          example: iam@gmail.com
+        userName:
+          type: string
+          example: iam
+        password:
+          type: string
+          example: iam123
+        properties:
+          type: array
+          example:
+            - key: body.scope
+              value: "true"
+            - key: mail.smtp.starttls.enable
+              value: true
+          items:
+            $ref: '#/components/schemas/Properties'
+    SMSSenderUpdateRequest:
+      required:
+        - providerURL
+        - provider
+        - contentType
+      type: object
+      properties:
+        provider:
+          type: string
+          example: NEXMO
+        providerURL:
+          type: string
+          example: https://rest.nexmo.com/sms/json
+        key:
+          type: string
+          example: 123**45
+        secret:
+          type: string
+          example: 5tg**ssd
+        sender:
+          type: string
+          example: +94 775563324
+        contentType:
+          type: string
+          enum:
+            - JSON
+            - FORM
+        properties:
+          type: array
+          example:
+            - key: http.headers
+              value: "X-Version: 1, Authorization: bearer ,Accept: application/json ,Content-Type: application/json"
+          items:
+            $ref: '#/components/schemas/Properties'
+    PushSenderUpdateRequest:
+      required:
+        - provider
+        - properties
+      type: object
+      properties:
+        provider:
+          type: string
+        properties:
+          type: array
+          example:
+            - key: fcm.serviceAccount
+              value: "jsonString"
+          items:
+            $ref: '#/components/schemas/Properties'
+    Properties:
+      required:
+        - key
+        - value
+      type: object
+      properties:
+        key:
+          type: string
+        value:
+          type: string
+      example:
+        - key: email
+          value: iam@gmail.com
+    EmailProviderList:
+      type: array
+      items:
+        $ref: '#/components/schemas/EmailSender'
+    SMSProviderList:
+      type: array
+      items:
+        $ref: '#/components/schemas/SMSSender'
+    PushProviderList:
+      type: array
+      items:
+        $ref: '#/components/schemas/PushSender'
+  parameters:
+    typeQueryParam:
+      name: type
+      in: query
+      description: |
+        Type of authenticators. Can be either 'LOCAL' or 'REQUEST_PATH'
+      required: false
+      style: form
+      explode: true
+      schema:
+        type: string
+  securitySchemes:
+    OAuth2:
+      type: oauth2
+      flows:
+        authorizationCode:
+          authorizationUrl: https://localhost:9443/t/{root-organization-name}/o/oauth2/authorize
+          tokenUrl: https://localhost:9443/t/{root-organization-name}/o/oauth2/token
```

Comment:
```suggestion
          authorizationUrl: https://localhost:9443/t/{root-organization-name}/oauth2/authorize
          tokenUrl: https://localhost:9443/t/{root-organization-name}/oauth2/token
```

Reply:
Fixed with https://github.com/wso2/docs-is/pull/5161/commits/d8a6c9c96654a492b94e6d7fb49f9243c440eafd